decidim-core/app/packs/src/decidim/account_form.js

```diff
@@ -24,4 +25,66 @@ $(() => {
       return false;
     });
   }
+
+
+  /**
+   * Resend the confirmation email logig here, we use fetch here because link is inside the form.
```

Comment:
I'm not a fan of using Javascript unless we have a good reason to do so. For me, it makes the feature much more complex that's needed. Can we implement this with plain Rails?

Reply:
(Maybe it's just my personal opinion, would like to hear @ahukkanen take on this one too)

Reply:
Yes, I agree @andreslucena .

I think we could simplify this a lot if we just added `data-remote` to the buttons and then implemented `nnn.js.erb` to handle the success/failure of the request.

Reply:
I didn't know that you can use data-remote links inside a form.. Refactored now